zerver/lib/message.py

```diff
@@ -306,18 +306,19 @@ def render_markdown(message, content, domain=None, realm_alert_words=None, messa
     else:
         message_user_ids = {u.id for u in message_users}
 
-    message.mentions_wildcard = False
-    message.is_me_message = False
-    message.mentions_user_ids = set()
-    message.alert_words = set()
-    message.links_for_preview = set()
-
-    if not domain:
-        domain = message.sender.realm.domain
-    if message.sending_client.name == "zephyr_mirror" and message.sender.realm.is_zephyr_mirror_realm:
-        # Use slightly customized Markdown processor for content
-        # delivered via zephyr_mirror
-        domain = u"zephyr_mirror"
+    if message is not None:
```

Comment:
To be able to render the content of the search without making a new `Message` object.